### PR TITLE
add styleTags to the deep dive documentation

### PR DESCRIPTION
**Pull request by JosefAssad**
ref summernote/summernote#2946

### CODE REVIEW

File: 01-deep-dive.md [hunk 1]
```diff
@@ -152,6 +152,33 @@ $('#summernote').summernote({
...
+  styleTags: [
+    'p',
+        { title: 'Blockquote', tag: 'blockquote', className: 'blockquote', value: 'blockquote' },
+        'pre', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6'
+	],
+  });
+{% endhighlight %}
+
+The tags can be specified just by tag name (as with `p` or `pre` or `h1`-`h6` above). It is also
+possible to customize the style in more defaul by providing an object looking like:
```
Comment (DiemenDesign) [inline 01-deep-dive.md]: You have a typo "defaul" == "default".
Good work on this PR, I'll tag the others so they can review it for merging.

**Comment by DiemenDesign** [comment]
Thank you for your contribution and interest in helping improve Summernote. I've added others to review your PR for merging. I think it's good to go though.

File: 01-deep-dive.md (hunk 1 above)
Comment (JosefAssad) [reply 01-deep-dive.md]: My bad, got it. :) It was actually "defaul" -> "detail". Thanks @DiemenDesign !

**Comment by DiemenDesign** [comment]
Thanks for your contribution @JosefAssad appreciate your efforts in improving the documentation.

**Pull request merged**